lib/validation/index.js

```diff
@@ -1,4 +1,11 @@
-import schema from './schema';
-import normalizeNode from './normalizeNode';
+import { childOfListIsAlwaysItem } from './childOfListIsAlwaysItem';
```

Comment:
How would you like to call the folder `normalizers` to match the naming with main codebase.